Makefile

```diff
@@ -134,7 +134,7 @@ push:
 
 .PHONY: git-commit
 git-commit:
-	git add catalog-manifests/
+	git add catalog-manifests/ manifests/
 	git commit -m "New catalog: $(CATALOG_VERSION)" --author="OpenShift SRE <aos-sre@redhat.com>"
```

Comment:
@jewzaam  Do we need to GPG sign these? I don't even know how that would work once we get to an automated pipeline.

Reply:
This isn't a new feature in the PR, happy to discuss in an issue.